dvc/tree/http.py

```diff
@@ -169,6 +172,18 @@ def get_file_hash(self, path_info):
 
         return HashInfo(self.PARAM_CHECKSUM, etag)
 
+    def _upload_fobj(self, fobj, to_info):
+        def chunks(fobj):
+            while True:
+                chunk = fobj.read(self.CHUNK_SIZE)
```

Comment:
There is a potential for timeout errors with 64mb chunk size, but if it was that size before we can keep it for now.